client/src/pages/ProfileList/List.tsx

```diff
@@ -89,47 +107,31 @@ export default function List(): JSX.Element {
             </Box>
           </Grid>
         </Grid>
-        <Grid container justify="center" spacing={6} className={`${classes.root}`}>
-          {profiles.map((profile, key) => {
-            const { firstName, lastName, user, description } = profile;
-            return (
-              user && (
-                <Grid item key={user}>
-                  <Card style={{ height: '100%' }} variant="outlined" key={key}>
-                    <CardActionArea component={Link} to={`profile/${user}`}>
-                      <Grid
-                        justify="center"
-                        alignItems="center"
-                        direction="column"
-                        container
-                        className={`${classes.card}`}
-                      >
-                        <Avatar src={profilePhoto} className={classes.avatar} />
-
-                        <CardContent>
-                          <Typography className={classes.fullName} variant="h6">
-                            {firstName} {lastName}
-                          </Typography>
-                          <Rating name="read-only" value={placeholder.rating} readOnly />
+        <Grid container spacing={10} className={`${classes.root}`} sm>
+          {profiles
+            .filter((profile: Profile) => {
+              if (profile.address && profile.address.includes(searchTerm)) {
+                return profile;
+              }
+            })
+            .filter((profile: Profile) => {
+              //TODO: Implement date search
+              // const requestDays = moment.range(new Date(selection.startDate), new Date(selection.endDate));
```

Comment:
You can implement this filtering logic outside of the JSX to improve readability 👍🏼 (e.g. 
```
const filteredProfileList = profiles.filter(...)
```
)

Reply:
I'm not 100% sure how to implement this to be honest. I'd love a bit of assistance if you can provide more information as this is something I've been wondering how to do. My issue is passing the props from filter to filter, it's not intuitive for me, maybe I'm missing something that would make it more clear.

Reply:
Oh, what I meant was to simlpy move this piece of logic outside of the `return` statement:
```
const filteredProfiles = profiles.filter(...).filter(...);
return; // clean jsx
```

I'm not sure I understand your question well but I think just chaining filters as you're doing should be fine. Can you elaborate on what the problem is exactly?

Reply:
Nothing anymore, I got it working after some fiddling!